content/en/docs/tasks/run-application/configure-pdb.md

```diff
@@ -229,6 +229,48 @@ status:
   observedGeneration: 1
 ```
 
+## Unhealthy Pod Eviction Policy
+
+{{< feature-state for_k8s_version="v1.26" state="alpha" >}}
+
+{{< note >}}
+In order to use this behavior, you must enable the `PDBUnhealthyPodEvictionPolicy`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+on the [API server](/docs/reference/command-line-tools-reference/kube-apiserver/).
+{{< /note >}}
+
+PodDisruptionBudget guarding an application ensures that `status.currentHealthy` number of pods
+does not fall under `status.desiredHealthy` number by not allowing eviction of healthy pods.
+By using `.spec.unhealthyPodEvictionPolicy`, you can also define the criteria when unhealthy pods 
+should be considered for eviction. The default behavior when no policy is specified corresponds 
+to the `IfHealthyBudget` policy.
+
+{{< note >}}
+Current implementation considers healthy pods, as pods that have `status.conditions` item with `type="Ready"` and `status="True"`.
+{{< /note >}}
```

Comment:
Do we want to define this in the doc? It is not mentioned anywhere else now.

Reply:
Yeah, probably a separate section which you could use in other places as links (see my other comment).

Reply:
created a new section under PDB status called `Healthiness of a Pod`